conf/default.php

```diff
@@ -3,3 +3,4 @@
 $conf['delay'] = 0;
 $conf['linkall_inclusions'] = '';
 $conf['linkall_exclusions'] = '';
+$conf['linkall_points_to'] = '';
```

Comment:
I'd probably call this `linkall_target_inclusions`, and create a `linkall_target_exclusions` to match.